dkim_keys.php

```diff
@@ -0,0 +1,167 @@
+<?php
```

Comment:
this does nothing related to dkims....it's just a copy of `ssl_certificates.php`

Reply:
yes - as I said. I wanted to use this as a starter for creating dkim-key show / delete page.